sot/opcode_translator/transform.py

```diff
@@ -1,11 +1,27 @@
 import dis
+import types
 
 from ..utils import log, log_do
 from .executor.opcode_executor import InstructionTranslatorCache
 from .skip_files import need_skip_path
 
 
-def eval_frame_callback(frame, **kwargs):
+def eval_frame_callback(
+    frame: types.FrameType, **kwargs
+) -> InstructionTranslatorCache | None:
+    """
+    Callback function for the frame evaluation process.
+    It will be executed when a frame is being translated.
+
+    Args:
+        frame(types.FrameType): The frame object representing the current code block.
```

Comment:
> current code block

当前代码块是什么呢？这里就是输入函数的 Frame Object 吧

```suggestion
        frame (types.FrameType): The frame object representing the current code block.
```